specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "NetworkManagementClient",
     "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
-    "version": "2017-03-30"
```

Comment:
What is the reason for this change?

Reply:
Ping @RajarajeshwariHS

Reply:
This was suggested by @number213 .

Reply:
Yes, if we're updating API version in `vmssPublicIpAddress`, we should also update it in `vmssNetworkInterface`. I guess the question is what's the reason for changing API version in VMSS specs? Just to use the latest one?

Reply:
We use latest Microsoft.Azure.Management.Network nuget which is of version “19.12.0-preview”. I searched in the source repository of this repo and found out the required API : ListVirtualMachineScaleSetPublicIPAddressesWithHttpMessagesAsync
https://github.com/Azure/azure-sdk-for-net/blob/dc23a39caf37d07f02512540d20c29866ff83759/sdk/network/Microsoft.Azure.Management.Network/src/Generated/PublicIpAddressesOperations.cs

Here the apiVersion is set as “2017-03-30”. We require it to be “2018-10-01”.
So, here https://github.com/Azure/azure-rest-api-specs/pull/6692 I have made the required changes to **vmssPublicIPAddress** and as we have to also update **vmssNetworkInterface**, I have make the changes here.

Reply:
@RajarajeshwariHS makes sense.
If you need updated Network SDK asap, we can work on getting you a signed nupkg, because the next NuGet release will be in a month